### PR TITLE
Fix the matchall in PHP proxy

### PR DESCRIPTION
**Pull request by cheetah90**

### CODE REVIEW

**Comment by cheetah90** [comment]
related to #189

File: PHP/proxy.php [hunk 1]
```diff
@@ -716,15 +716,15 @@ public function canProcessRequest()
 
             //check with listed serverurl regardless if mustMatch is true or false
             foreach ($this->serverUrls as $key => $value) {
-                
                 $serverUrl = $value['serverurl'][0];
                 $serverUrl['url'] = $this->sanitizeUrl($serverUrl['url']); //Do all the URL cleanups and checks at once
-                
+				$serverUrl['matchall'] = strtolower((string) $serverUrl['matchall']);
```
Comment (bsvensson) [inline PHP/proxy.php]: indentation?

File: PHP/proxy.php [hunk 2]
```diff
@@ -716,15 +716,15 @@ public function canProcessRequest()
...
+
```
Comment (bsvensson) [inline PHP/proxy.php]: I think the following 3 lines can be deleted with the `strtolower((string)...` change above.

File: PHP/proxy.php (hunk 1 above)
Comment (cheetah90) [inline PHP/proxy.php]: fixed

File: PHP/proxy.php (hunk 2 above)
Comment (cheetah90) [inline PHP/proxy.php]: deleted

**Comment by bsvensson** [comment]
Verified with PHP 5.3 on IIS/Windows 7 and PHP v5.5.14 on Apache/Mac.

**Pull request merged**